executor/new_distsql.go

```diff
@@ -125,6 +132,11 @@ func (e *TableReaderExecutor) Next() (Row, error) {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		if !e.aggregate && e.genValues != nil {
+			if evalGeneratedColumns(e.columns, e.genValues, values) != nil {
```

Comment:
I think this way is not elegent, can we add a `Projection` operator to do this ?